### DISCUSSION TITLE
Split SIG-Core into more SIG's

### DISCUSSION THREAD

**Discussion by amzn-pratikpa** [o3de/sig-core#16] [Q&A]
For the SIG-Core meeting on 15-OCT-2021, a point was raised that SIG-Core should be split into other SIG's. (Notes are here: https://github.com/o3de/sig-core/blob/main/meetings/notes/SIG-Core-Notes%20-%202021-OCT-15.md)

The reason for this is that Core contains too many areas of responsibility and it's hard to keep focus. The following new groups were suggested:

* SIG-Physics
* SIG-Animation
* SIG-AI
Or combine the above 3 SIG's into:
* SIG-Simulation (AI (future), Physics, Simulation)
      
* SIG-Systems: which is everything currently in core minus the above. Note that this could still be named just SIG-Core

* SIG-Scripting: scripting is currently in both content and core and would move to this discrete SIG, taking responsibility for Script Canvas, Lua, Python
 

***Please comment with your thoughts on the proposal to split the SIG and what, if any, the other SIG's should be.***

**Comment by bosnichd** [comment]
I think SIG-Simulation, SIG-Systems, and SIG-Scripting is a good starting point, then SIG-Simulation can decide if they want to further break up into SIG-Physics, SIG-Animation, and SIG-AI

While we're at it, we talked a while ago about whether input should move to sig-platform, and I think the general consensus at the time was that it should, but I don't think it ever happened. The fact that all the discussions about adding mouse/keyboard/gamepad support for Linux are happening in sig-platform is good evidence that it should indeed probably be moved there:)
